provider/datagen/src/source.rs

```diff
@@ -282,6 +291,47 @@ impl TomlCache {
     }
 }
 
+pub(crate) struct JsonCache {
```

Comment:
Please make `transform::cldr::source::CldrCache` use this, it's almost the same code.